### DISCUSSION TITLE
Vipps OAuth Provider

### DISCUSSION THREAD

**Discussion by muffeeee** [supabase/supabase#14093] [Feature Requests]
Vipps is the no. 1 mobile payment provider in Norway - in fact, it has become the de facto mobile payment provider in the country, with basically no competition. They provide a service called Vipps Login, where users can authenticate themselves via the Vipps app on their phone. This authentication method is used for a ton of Norwegian websites, and it would be really helpful for us norwegian devs if Supabase would provide us with a Vipps provider!

Vipps Login mostly adheres to the OpenID and OAuth 2.0 standards.
There is pretty good documentation available (in English!) here: https://developer.vippsmobilepay.com/docs/APIs/login-api/api-guide/integration/

I know this will probably get less prioritized as it's purely a Norwegian thing, but I'm hoping for the best. If anything, I can be of help in testing the integration (although Vipps also definitely responds to emails from developers). TIA!

**Comment by jonkristian** [comment]
I've added this as a request here: https://github.com/supabase/auth/issues/451 , go give it a :+1:
